### PR TITLE
add 'pulled from/into wrong branch'

### PR DESCRIPTION
**Pull request by joshdutcher**
my text editor automatically trims trailing spaces, so that happened too I guess.

### CODE REVIEW

**Comment by RichardLitt** [comment]
Could you remove the lines with the trailing spaces and squash this commit? 

I'm not sure about the credit line - it's not like this isn't publicly available information in the git man, anyway. What do you think, @k88hudson

**Comment by k88hudson** [comment]
Sorry for the delay @joshdutcher, thanks for doing this.

Personally I think this is standard procedure enough to omit the credit line, but I'm open to either way.

That said, if larger sections are used or need to be linked to, adding something to a "resources" section makes sense to me.

**Comment by joshdutcher** [comment]
I'm totally fine with removing the credit line, and I'll squash the commit.

File: README.md [hunk 1]
```diff
@@ -161,11 +161,9 @@ Changes not staged for commit:
 In this example, `README.md` has conflicts. Open that file and look for the following:
 
 ```
-<<<<<<< HEAD
-some code
-=========
+
 some code
->>>>>>> new-commit
+
```
Comment (RichardLitt) [inline README.md]: These lines shouldn't have been edited - the rest looks good. If you can add these back in, we're ready to go.

**Comment by joshdutcher** [comment]
Sorry about that - got the lines back in there.

**Comment by k88hudson** [comment]
@joshdutcher I did the same thing during a rebase hehe

**Comment by RichardLitt** [comment]
Landed! Thanks @joshdutcher

**Comment by joshdutcher** [comment]
Thank you. I use git lots at work but this was my first experience contributing to a project on github, so I appreciate the patience and guidance.

**Comment by k88hudson** [comment]
Awesome! Thanks so much for contributing :+1:

**Comment by RichardLitt** [comment]
Oh man, welcome! Thanks @joshdutcher

**Pull request merged**